### PR TITLE
Show dashes (-) instead of 0 in the table

### PR DESCRIPTION
**Pull request by GordonBeeming**
### Motivation

Show dashes (-) instead of 0 in the table
#442 

### Test Types

- [x] Unit tests
- [ ] Manual tests

### CODE REVIEW

**Comment by GordonBeeming** [comment]
@microsoft-github-policy-service agree

**Comment by GordonBeeming** [comment]
This should be good to accept now 😅

**Comment by GordonBeeming** [comment]
@muiriswoulfe do you mind taking a look at this PR, and see if it's something you'd want in the action 🙂

File: src/task/src/pullRequests/pullRequestComments.ts [hunk 1]
```diff
@@ -202,6 +202,11 @@ export default class PullRequestComments {
       surround = '**'
     }
 
-    return `${surround}${title}${surround}|${surround}${metric.toLocaleString()}${surround}\n`
+    let metricString = metric.toLocaleString();
```
Comment (muiriswoulfe) [inline src/task/src/pullRequests/pullRequestComments.ts:205]: Could you follow the linter requirements and remove the `;`s?

File: src/task/task.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "$schema": "https://raw.githubusercontent.com/microsoft/azure-pipelines-task-lib/master/tasks.schema.json",
   "id": "907d3b28-6b37-4ac7-ac75-9631ee53e512",
   "name": "PRMetrics",
-  "friendlyName": "PR Metrics v1.5.9",
+  "friendlyName": "PR Metrics v1.5.10",
```
Comment (muiriswoulfe) [inline src/task/task.json:5]: Please revert this change and the other version update. I'll be updating the project in January via an automated process, so I'd prefer to hold off for a full update until then

**Review by muiriswoulfe — CHANGES_REQUESTED** [review]
Thanks for your changes. Overall, this looks good, but I've just left a few small comments.

**Comment by GordonBeeming** [comment]
Thanks @muiriswoulfe, I've done the updates requested 😊

File: src/task/task.json (hunk 2 above)
Comment (GordonBeeming) [reply src/task/task.json]: FYI: I had updated this just from the contribution guide saying we should 🙂

https://github.com/microsoft/PR-Metrics/blob/main/.github/CONTRIBUTING.md#updating-an-extension

**Comment by GordonBeeming** [comment]
It looks like there's token issues at the moment 😓

**Comment by muiriswoulfe** [comment]
> It looks like there's token issues at the moment 😓

The issue is actually this one – it's a known issue with GitHub: https://github.com/openmainframeproject/feilong/issues/367

I'll need to create a local copy to validate this.

**Comment by GordonBeeming** [comment]
@GordonBeeming Done - Thanks ☺️

**Review by muiriswoulfe — APPROVED** [review]

**Pull request merged**